README.md

```diff
@@ -35,6 +35,14 @@ It provides the following interfaces:
     cmake -DQt5_DIR:PATH=/path/to/QtX.Y.Z/X.Y.Z/gcc_64/lib/cmake/Qt5 ../qRestAPI
     make -j4
 
+## How to build using Qt 6.x
```

Comment:
In addition to this, @kislinsk could you add Qt6 to the prerequisites section in this doc?
` * [Qt](https://www.qt.io/) 4.x or 5.x`